app/code/core/Mage/Eav/Model/Entity/Attribute/Frontend/Abstract.php

```diff
@@ -89,16 +89,6 @@ public function getLabel()
         return $label;
     }
 
-    /**
```

Comment:
As explained in the comment to the first commit I remove this function because it makes no sense to put it in the "Frontend" class. It is outside the business logic.